internal/translation/ipaccesslist/conversion.go

```diff
@@ -0,0 +1,192 @@
+package ipaccesslist
+
+import (
+	"fmt"
+	"net"
+	"strings"
+	"time"
+
+	"go.mongodb.org/atlas-sdk/v20231115008/admin"
+
+	"github.com/mongodb/mongodb-atlas-kubernetes/v2/internal/timeutil"
+	"github.com/mongodb/mongodb-atlas-kubernetes/v2/pkg/api/v1/project"
+)
+
+type IPAccessEntry struct {
+	CIDR             string
+	AWSSecurityGroup string
+	DeleteAfterDate  *time.Time
+	Comment          string
+}
+
+func (i *IPAccessEntry) ID() string {
+	if i.CIDR != "" {
+		return i.CIDR
+	}
+
+	return i.AWSSecurityGroup
+}
+
+func (i *IPAccessEntry) IsExpired() bool {
+	if i.DeleteAfterDate == nil {
+		return false
+	}
+
+	return i.DeleteAfterDate.Before(time.Now())
+}
+
+type IPAccessEntries map[string]*IPAccessEntry
+
+func (i IPAccessEntries) GetActives() IPAccessEntries {
```

Comment:
this method can be removed in favor of a single `ByExpired(expired bool)` method (see below)